package.json

```diff
@@ -15,7 +15,7 @@
     "eslint": "^3.19.0",
     "fuzzer": "^0.2.1",
     "gh-pages": "^0.12.0",
-    "jsdoc": "git+https://github.com/jvanveen/jsdoc.git#a789c902c56e7852ecff1825dc4f5d8fcd36a5ed",
+    "jsdoc": "^3.5.4",
```

Comment:
Awesome to see JSDoc update the parser (https://github.com/jsdoc3/jsdoc/blob/master/CHANGES.md#354-august-2017).